app/utils/PagerDutyHelper.scala

```diff
@@ -32,6 +32,7 @@ object PagerDutyHelper {
     final val FAILED_TO_INSERT_FILE_NOTIFICATION = Value
     final val NOTIFICATION_SET_TO_PERMANENT_FAILURE = Value
     final val NOTIFICATION_SET_TO_NOT_PROCESSED_PENDING_RETRY = Value
+    final val NOTIFICATION_SET_TO_NOT_RECEIVED_IN_SDES_PENDING_RETRY = Value
```

Comment:
Is there an alert config PR for this?

Reply:
Raised https://github.com/hmrc/alert-config/pull/2974